### ISSUE TITLE
Integrate leaderboard into app

### ISSUE THREAD

**Issue by laurentS** [mapswipe/mapswipe#178]
Look into ways to include the leaderboard or a portion relevant to the logged in user into the app.

**Comment by els2171** [comment]
Agreed! It would be awesome even to just see what percentile you're in next to your badge.

**Comment by danbjoseph** [comment]
The community has made efforts to provide information to users about their contributions while trying to avoid gamification aspects that might encourage speed over accuracy. Such considerations factored into the design of the [Community Pages](https://community.mapswipe.org) (which replaced the v1 leaderboard).

**Comment by els2171** [comment]
That makes a lot of sense! Thanks for sharing!
